### ISSUE TITLE
TypeError: Cannot read property 'keybase' of undefined

### ISSUE THREAD

**Issue by faboweb** [cosmos/voyager#1211]
https://sentry.io/tendermint/cosmos-voyager/issues/667195282/

```
TypeError: Cannot read property 'keybase' of undefined
  at Store.setKeybaseIdentity (/private/var/folders/m9/14ny26h57vj1kt0_twc07ydc0000gn/T/AppTranslocation/472A41FE-40CF-4336-B823-C9C428E4EAD8/d/Cosmos%20Voyager.app/Contents/Resources/app/dist/renderer.js:106905:22)
  at wrappedMutationHandler (/private/var/folders/m9/14ny26h57vj1kt0_twc07ydc0000gn/T/AppTranslocation/472A41FE-40CF-4336-B823-C9C428E4EAD8/d/Cosmos%20Voyager.app/Contents/Resources/app/dist/renderer.js:6646:13)
  at commitIterator (/private/var/folders/m9/14ny26h57vj1kt0_twc07ydc0000gn/T/AppTranslocation/472A41FE-40CF-4336-B823-C9C428E4EAD8/d/Cosmos%20Voyager.app/Contents/Resources/app/dist/renderer.js:6339:7)
  at Array.forEach (<anonymous>)
  at None (/private/var/folders/m9/14ny26h57vj1kt0_twc07ydc0000gn/T/AppTranslocation/472A41FE-40CF-4336-B823-C9C428E4EAD8/d/Cosmos%20Voyager.app/Contents/Resources/app/dist/renderer.js:6338:11)
...
(10 additional frame(s) were not displayed)
```

**Comment by NodeGuy** [comment]
What are the steps to reproduce this?

**Comment by faboweb** [comment]
Check Sentry. I have no idea. But it was reported by our error logger.

**Comment by faboweb** [comment]
Just happened to me. Apparently happens when you sign out while the keybase data is still loading. When signing out the delegate list is cleared. So it can't find the delegate anymore, it wanted to load the keybase data for.

**Comment by faboweb** [comment]
Will be fixed with #1216

**Comment by sabau** [comment]
![screenshot 2019-03-01 at 12 59 47](https://user-images.githubusercontent.com/1656471/53636930-0b70b080-3c22-11e9-8cd4-e8d6a819cf39.png)
